aws_helper/config.go

```diff
@@ -107,6 +113,38 @@ func CreateAwsSessionFromConfig(config *AwsSessionConfig, terragruntOptions *opt
 	return sess, nil
 }
 
+type tokenFetcher string
+
+// Implements the stscreds.TokenFetcher interface
+// Supports providing a token value or the path to a token on disk
+func (f tokenFetcher) FetchToken(ctx credentials.Context) ([]byte, error) {
```

Comment:
Comment should have the following format 'FetchToken ...'

Reply:
Done.